src/recipes/Login.tsx

```diff
@@ -499,6 +500,7 @@ export default class Login extends React.PureComponent<LoginProps, LoginState> {
                       ? isButtonDisabled
                       : !isTenantValid
                   }
+                  loading={isSubmitButtonLoading}
```

Comment:
check if this fixes the problem in both login flows: old and new

Reply:
checked on both